.jenkinsci-new/utils/vars.groovy

```diff
@@ -10,7 +10,7 @@
 
 def compilerMapping () {
   return ['gcc5': ['cxx_compiler':'g++-5', 'cc_compiler':'gcc-5'],
-          'gcc7' : ['cxx_compiler':'g++-7', 'cc_compiler':'gcc-7'],
+          'gcc7' : ['cxx_compiler':"${WORKSPACE}/.jenkinsci-new/helpers/build.sh", 'cc_compiler':'gcc-7',],
```

Comment:
Default compiler is gcc5, in gcc7 it builds just last build before merge to trunk(starts by hand)